lib/types/terminus.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+var types = require('../types'),
+    Terminus = module.exports = {};
+
+Terminus.Source = types.defineComposite({
+  name: 'source', code: 0x28,
+  fields: [
+    { name: 'address', type: 'string' },
```

Comment:
address is another one where it's technically '*' but I've never seen anything but string. Comment to that effect?